helm_deploy/laa-estimate-eligibility/values/ccq-uat.yaml

```diff
@@ -0,0 +1,112 @@
+# Default values for laa-check-client-qualifies-uat.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 2
+
+resources: {}
+
+image:
+  repository: nginx
+  tag: stable
+  pullPolicy: IfNotPresent
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+nodeSelector: {}
+
+tolerations: []
+
+affinity: {}
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: false
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: "cd-serviceaccount"
+
+podSecurityContext: {}
+# fsGroup: 2000
+
+service:
+  type: ClusterIP
+  port: 80
+
+securityContext:
+  runAsUser: 1000
+
+ingress:
+  hosts:
+#    - host: main-check-client-qualifies-legal-aid-uat.cloud-platform.service.justice.gov.uk
+    - host: main-ccq-uat.cloud-platform.service.justice.gov.uk
```

Comment:
My assumption is that we can't use the actual main UAT URL as the host name here because it's already in use in the old namespace and something will break if we declare the same host for both. But did we actually test that? If the weight is set to 0 here, would it actually _allow_ us to have both namespaces declare the same host?

I'm asking because we need to solve this problem at some point, and I've got solving this problem earmarked for EL-1214 which on JIRA is marked as blocking this ticket. If you've learned anything about this as part of this ticket, I'm eager to see if we can use it to make progress on EL-1214.

Reply:
no we did not test it because testing it could only happen after merging the PR so we are playing it safe by changing it for now. Once we can dual deploy we can run a branch to test exactly how it works, as this will apply to UAT/STG and PROD envs